cmd/kube-scheduler/app/server.go

```diff
@@ -115,7 +115,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.policyConfigFile, "policy-config-file", o.policyConfigFile, "File with scheduler policy configuration. This file is used if policy ConfigMap is not provided or --use-legacy-policy-config==true")
 	usage := fmt.Sprintf("Name of the ConfigMap object that contains scheduler's policy configuration. It must exist in the system namespace before scheduler initialization if --use-legacy-policy-config==false. The config must be provided as the value of an element in 'Data' map with the key='%v'", componentconfig.SchedulerPolicyConfigMapKey)
 	fs.StringVar(&o.policyConfigMapName, "policy-configmap", o.policyConfigMapName, usage)
-	fs.StringVar(&o.policyConfigMapNamespace, "policy-configmap-namespace", o.policyConfigMapNamespace, "The namespace where policy ConfigMap is located. The system namespace will be used if this is not provided or is empty.")
+	fs.StringVar(&o.policyConfigMapNamespace, "policy-configmap-namespace", o.policyConfigMapNamespace, "The namespace where policy ConfigMap is located. The kube-system namespace will be used if this is not provided or is empty.")
```

Comment:
how about this?

```
-       fs.StringVar(&o.policyConfigMapNamespace, "policy-configmap-namespace", o.policyConfigMapNamespace, "The namespace where policy ConfigMap is located. The system namespace will be used if this is not provided or is empty.")
+       fs.StringVar(&o.policyConfigMapNamespace, "policy-configmap-namespace", metav1.NamespaceSystem, "The namespace where policy ConfigMap is located. The system namespace will be used if this is not provided or is empty.")
```

Reply:
Quoted from @deads2k 

> Binding flags to structs allows the structs to be easily unit tested with predictable default behavior. Binding them to structs and using the current value as the default value allows the defaults to be manipulated in a `NewFooOptions` method for consistent use in those unit tests. It also allows for future embedding which is very useful for generic things.

I prefer `o.policyConfigMapNamespace`, instead of `metav1.NamespaceSystem`.

/cc @deads2k

Reply:
> I prefer o.policyConfigMapNamespace, instead of metav1.NamespaceSystem.

agree. it is unintuitive that the act of binding the flag to the struct field would change its current value.

Reply:
> easily unit tested

Reasonable to me :)

 @zjj2wry , sorry for that, would you help to address the comments? I'd like to get this merged :)

Reply:
ok, I think it can be used as a specification in the future.

@k82cn addressed ：）